app/settings.py

```diff
@@ -66,7 +66,24 @@
 OSF_BASE_URL = env.OSF_BASE_URL.rstrip("/")
 OSF_API_BASE_URL = env.OSF_API_BASE_URL.rstrip("/")
 ALLOWED_RESOURCE_URI_PREFIXES = {OSF_BASE_URL}
-SESSION_COOKIE_DOMAIN = env.SESSION_COOKIE_DOMAIN
+# SESSION_COOKIE_DOMAIN = env.SESSION_COOKIE_DOMAIN
```

Comment:
I couldn't find `SESSION_COOKIE_DOMAIN` in the code base, so maybe we can simply remove it?